File: samples/java/TicTacToe/Client/src/main/java/systems/zlink/samples/tictactoe/client/TicTacToeSampleDefaults.java
package systems.zlink.samples.tictactoe.client;

import java.time.Duration;

public final class TicTacToeSampleDefaults {
    public static final String ApiUrl = "http://127.0.0.1:18080";
    public static final String GameName = "tictactoe-game";
    public static final String XActorId = "player-x";
    public static final String OActorId = "player-o";
    public static final String ObserverActorId = "observer";
    public static final Duration RequestTimeout = Duration.ofSeconds(10);

    private TicTacToeSampleDefaults() {}
}

File: samples/kotlin/TicTacToe/Client/src/main/kotlin/systems/zlink/samples/kotlin/tictactoe/client/TicTacToeSampleDefaults.kt
package systems.zlink.samples.kotlin.tictactoe.client

import java.time.Duration

object TicTacToeSampleDefaults {
    const val ApiUrl: String = "http://127.0.0.1:18081"
    const val GameName: String = "tictactoe-game"
    const val XActorId: String = "player-x"
    const val OActorId: String = "player-o"
    const val ObserverActorId: String = "player-observer"
    val RequestTimeout: Duration = Duration.ofSeconds(10)
}
